libs/core/src/lib/components/wrapper-component.ts

```diff
@@ -142,9 +142,11 @@ export abstract class ReactWrapperComponent<TProps extends {}> implements AfterV
     this._shouldSetHostDisplay = setHostDisplay;
   }
 
-  ngAfterViewInit() {
+  ngAfterContentInit() {
```

Comment:
Why the change?
Also, if this stays - make sure to change the `implements` of this component respectively.

Reply:
I am actually hoping for your insight on this. The following conversation applies to the Calendar and ComboBox.

The component was not binding properties from the directive properly when the passAttributesAsProps call was in ngAfterContentInit, but it would only do this in the specific case where the binding was only coming from the directive, and not from an attribute. If we also set the property directly through attribute-binding, then Angular binds the React properties correctly. This is not ideal, sort of messy, and not how anyone would reasonably use the component.

After some investigation, Jerry and I came to the conclusion that it may be a timing issue. If the passAttributesAsProps function call happened early and late (in ngOnChanges, which would happen with attribute-binding), then everything is fine. If the call just happened late (in ngAfterContentInit), then the binding would not happen correctly, and the event callbacks would fire with incorrect code. Moving the passAttributesAsProps function call to happen earlier in the Angular lifecycle fixed the problem.

Reply:
I'm not sure I understand why passing it later than you can (i.e. in anything later than `ngOnChanges`) needed is an issue.

Reply:
ngOnChanges doesn't fire consistently.

Reply:
Right, but what about the difference between `ngAfterViewInit` vs `ngAfterContentInit` in this case?

Reply:
Firing passAttributesAsProps at ngAfterViewInit time works. Firing it at ngAfterContentInit causes event bindings to bind incorrectly.